windows/Win32/Main.cpp

```diff
@@ -50,9 +50,12 @@ namespace
 
         auto hwnd = winrt::Microsoft::UI::GetWindowFromWindowId(window.Id());
         auto scaleFactor = ScaleFactor(hwnd);
+        winrt::Microsoft::ReactNative::LayoutConstraints constraints;
         winrt::Size size{window.ClientSize().Width / scaleFactor,
                          window.ClientSize().Height / scaleFactor};
-        rootView.Arrange({size, size, winrt::LayoutDirection::Undefined}, {0, 0});
+        constraints.MinimumSize = constraints.MaximumSize = size;
+        constraints.LayoutDirection = winrt::LayoutDirection::LeftToRight;
```

Comment:
This is really odd. I looked at `react-native-windows` and cannot see `LayoutDirection` being set at all: https://github.com/search?q=repo%3Amicrosoft%2Freact-native-windows+UpdateRootViewSizeToAppWindow&type=code

How does this work upstream? I'm not sure hard-coding left-to-right is the correct thing to do here.

Reply:
@acoates: What's the right fix here? It looks like the playground app hard-codes it to `winrt::LayoutDirection::LeftToRight`, but should it be read from somewhere?